core/spec.md

```diff
@@ -4035,13 +4035,15 @@ and the following Resource level attributes:
   Implementations MUST include `none` as one of the possible values and when
   set to `none` then compatibility checking MUST NOT be performed.
 
-  If the `enforcecompatibility` capability is set to `true` then
+  If the `compatibilityauthority` attribute is set to `server` then
   implementations of this specification are REQUIRED to perform the proper
   compatibility checks to ensure that all Versions of a Resource adhere to the
-  rules defined by the current value of this attribute. This includes
-  generating an error in response to requests to add/delete/modify Versions or
-  update this attribute in such a way as to make the existing Versions
-  incompatible due to the new value.
+  rules defined by the current value of this attribute.
+  For `compatibility` strategies that require understanding the sequence in
+  which versions need to be compatible, the server MUST use the [`ancestor`]
```

Comment:
v->V
I was wondering if you got them all :-)

Reply:
![image](https://github.com/user-attachments/assets/96192cd3-f419-4662-8a03-a7fe70689f43)